package.json

```diff
@@ -57,6 +57,7 @@
     "less": "2.7.1",
     "less-loader": "2.2.3",
     "postcss-loader": "0.9.1",
+    "prop-types": "^15.5.10",
```

Comment:
I wondered the same thing, but Facebook recommend that you install it as a regular dependency: https://github.com/facebook/prop-types/blob/master/README.md#how-to-depend-on-this-package